### PR TITLE
Fix error message in identifiers extractor

### PR DESCRIPTION
**Pull request by Nek-**
It was probably a bad copy/paste

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | None
| License       | MIT
| Doc PR        | N/A

### CODE REVIEW

**Comment by soyuka** [comment]
tests need to be updated also

File: src/Api/IdentifiersExtractor.php [hunk 1]
```diff
@@ -42,7 +42,7 @@ public function __construct(PropertyNameCollectionFactoryInterface $propertyName
         $this->resourceClassResolver = $resourceClassResolver;
 
         if (null === $this->resourceClassResolver) {
-            @trigger_error(sprintf('Not injecting %s in the CachedIdentifiersExtractor might introduce cache issues with object identifiers.', ResourceClassResolverInterface::class), E_USER_DEPRECATED);
+            @trigger_error(sprintf('Not injecting %s in the IdentifiersExtractor might introduce cache issues with object identifiers.', ResourceClassResolverInterface::class), E_USER_DEPRECATED);
```
Comment (teohhanhui) [inline src/Api/IdentifiersExtractor.php]: It could be changed to use `__CLASS__` instead? And remove "the".

Comment (Nek-) [reply src/Api/IdentifiersExtractor.php]: Is it a must have?

**Comment by Nek-** [comment]
CI didn't start or...?

**Comment by soyuka** [comment]
Thanks @Nek-

**Pull request merged**